src/components/common/TmBalance.vue

```diff
@@ -144,6 +144,14 @@
               <i class="material-icons">send</i></button
             ><span>Send</span>
           </div>
+          <div v-if="developmentMode" class="icon-button-container">
```

Comment:
change to experimental mode?

Reply:
oh, ok. It is easier to test like this though. But ok

Reply:
👀 ?? What is the difference between experimental and developmentMode currently? In `session.js` I see:

```
    developmentMode: config.development, // can't be set in browser
    experimentalMode: config.development, // development mode, can be set from browser
```